src/test/java/com/spotify/docker/client/jackson/UnixTimestampSerializerTest.java

```diff
@@ -0,0 +1,60 @@
+/*-
+ * -\-\-
+ * docker-client
+ * --
+ * Copyright (C) 2016 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.docker.client.jackson;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.spotify.docker.client.ObjectMapperProvider;
+
+import java.util.Date;
+
+import org.junit.Test;
+
+public class UnixTimestampSerializerTest {
+
+  private static final ObjectMapper OBJECT_MAPPER =
+      new ObjectMapperProvider().getContext(UnixTimestampSerializerTest.class);
```

Comment:
The `ObjectMapperProvider` is a bit silly. The `Json` class here is better https://github.com/spotify/helios/blob/master/helios-client/src/main/java/com/spotify/helios/common/Json.java because it's all static methods using a private ObjectMapper delegate.